rules/microsoft_graph/microsoft_graph_passthrough.py

```diff
@@ -0,0 +1,22 @@
+def rule(event):
+    return event.get("status") == "newAlert"
+
+
+def title(event):
+    return f"Microsoft Graph Alert ({event.get('title')})"
+
+
+def dedup(event):
+    return event.get("id")
+
+
+def severity(event):
+    return event.get("severity")
+
+
+def alert_context(event):
+    return {
```

Comment:
This seems like a good place to use the global helper

Reply:
I was going to modify it afterwards to use the helper since helpers are updated with panther releases. There's a race condition if someone pulls new detections before the helpers are updated they would error.